src/Controls/src/Core/Compatibility/Handlers/TableView/Android/TableViewRenderer.cs

```diff
@@ -107,7 +137,10 @@ public override SizeRequest GetDesiredSize(double widthConstraint, double height
 							continue;
 						}
 
-						AView listItem = _adapter.GetView(i, null, Control);
+						var platformView = cell.Handler?.PlatformView as AView;
```

Comment:
Should this be ToPlatform()? If we don't want to create, will it break if it is using a wrapper/container view? Or is that not possible with these views